custom_components/modernforms/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-SCAN_INTERVAL = timedelta(seconds=5)
+SCAN_INTERVAL = 5
```

Comment:
So, to be practical, as this still concerns me. Let's roll this back to 60, and focus on other changes in this PR, because they seem to address two slightly different issues observed with the fans.

@jimpastos is, obviously, final decision maker on this one.

Reply:
I agree 30s or 60s is ideal. Anything lower than that should be configurable explicitly by users